ansible/roles/sunbird-system-init/templates/system_init_commands.j2

```diff
@@ -0,0 +1,344 @@
+#!/bin/sh
+
+validateConfigFields() {
+        if [ -z "{{sunbird_custodian_tenant_name}}" ] || [ -z "{{sunbird_custodian_tenant_description}}" ] || \
+           [ -z "{{sunbird_custodian_tenant_channel}}" ] || [ -z "{{sunbird_root_user_firstname}}" ] || \
+           [ -z "{{sunbird_root_user_username}}" ] || [ -z "{{sunbird_root_user_password}}" ] || \
+           [ -z "{{sunbird_root_user_email}}" ] || [ -z "{{sunbird_root_user_phone}}" ];
+        then
+                echo "System Initialisation failed. One or more mandatory variables are not configured(sunbird_custodian_tenant_name,"\
```

Comment:
Can we display the exact variable which is not configured so that there is no ambiguity for the user running the script.

Reply:
Review Comment Incorporated in code